src/content-helper/editor-sidebar/performance-details/component.tsx

```diff
@@ -1,50 +1,198 @@
 /**
  * WordPress dependencies
  */
-import { Button, Spinner } from '@wordpress/components';
+import { Button, MenuGroup, MenuItem, SelectControl } from '@wordpress/components';
 import { useEffect, useState } from '@wordpress/element';
-import { __, _n, sprintf } from '@wordpress/i18n';
+import { __, sprintf } from '@wordpress/i18n';
+import {
+	check,
+	moreVertical,
+	reset,
+} from '@wordpress/icons';
+import { Telemetry } from '../../../js/telemetry/telemetry';
 
 /**
  * Internal dependencies
  */
 import { ContentHelperError } from '../../common/content-helper-error';
-import { Period, getPeriodDescription } from '../../common/utils/constants';
-import { formatToImpreciseNumber } from '../../common/utils/number';
+import { SidebarSettings, useSettings } from '../../common/settings';
+import {
+	Period,
+	getPeriodDescription,
+	isInEnum,
+} from '../../common/utils/constants';
+import { PerformanceStatPanel } from './component-panel';
+import { PerformanceCategoriesPanel } from './component-panel-categories';
+import { PerformanceOverviewPanel } from './component-panel-overview';
+import { PerformanceReferrersPanel } from './component-panel-referrers';
 import { PerformanceData } from './model';
 import { PerformanceDetailsProvider } from './provider';
 
 // Number of attempts to fetch the data before displaying an error.
 const FETCH_RETRIES = 1;
 
 /**
- * Defines the props structure for PerformanceDetails.
+ * List of available panels to display in the Performance Stats menu.
  *
- * @since 3.11.0
+ * @since 3.14.0
  */
-interface PerformanceDetailsProps {
-	period: Period;
-}
+const availablePanels = [
+	{
+		name: 'overview',
+		label: __( 'Overview', 'wp-parsely' ),
+		forced: true,
```

Comment:
Is this `forced` due to any technical constraint or issue?

Reply:
'forced' means that the panel cannot be disabled. There isn't any technical reason, other than avoiding an empty performance tab if all the panels are disabled. @dabowman, any thoughts on this one?

Reply:
It's still possible to hide all data points, ending up with an empty performance tab. Although technically possible, it doesn't make a lot of sense from a user perspective to just empty everything. We could probably introduce one or more empty states in the future, or give users to get rid of the tab if they don't want it altogether.

I'd probably remove the `forced` property here, just in case there's a legit use case for hiding the overview.

Reply:
Sounds good, I will remove the `forced`. I will keep the logic, in case we want to use it for something else in the future, but will remove it from the overview panel.